src/js/components/AutolinkComponent.jsx

```diff
@@ -0,0 +1,56 @@
+var React = require("react/addons");
+
+/**
+ * Credits: https://github.com/banyan/react-autolink
+ * Lifting off some code as the package is not compatible with React 0.13
+ * TODO remove and add to dependecies
+ * after https://github.com/mesosphere/marathon/issues/2710
+ */
+
+const URLDelimiter = /(https?:\/\/[^\s]+)/gi;
+
+var AutolinkComponent = React.createClass({
+  displayName: "AutolinkComponent",
+
+  propTypes: {
+    options: React.PropTypes.object,
+    text: React.PropTypes.string.isRequired
+  },
+
+  getDefaultProps: function () {
+    return {
+      options: {
+        target: "_blank"
+      }
+    };
+  },
+
+  autolink: function () {
+    return this.props.text
+      .split(URLDelimiter)
+      .map(word => {
+        var match = word.match(URLDelimiter);
+        if (match) {
+          let url = match[0];
+
+          return React.createElement("a",
+            Object.assign({}, {
+              key: `autolink-${url}`,
+              href: url
+            }, this.props.options),
+            url
+          );
+        } else {
```

Comment:
`else` not needed, because of the `return` above.

Sorry for commenting late, just noticed.